drivers/CoreHTS/tests/CoreHTS_test.cpp

```diff
@@ -35,7 +35,7 @@ class CoreHTSTest : public ::testing::Test
 	const uint8_t i2c_address {0xBF};
 
 	CoreHTS corehts;
-	CoreI2CMock mocki2c;
+	mock::CoreI2C mocki2c;
```

Comment:
Si on évite l'utilisation du terme mock dans le nom de l'objet

```suggestion
	mock::CoreI2C i2c;
```
ou
```suggestion
	mock::CoreI2C corei2c;
```